nucypher/blockchain/eth/actors.py

```diff
@@ -1086,6 +1095,28 @@ def disable_winding_down(self) -> dict:
         receipt = self._set_winding_down(value=False)
         return receipt
 
+    def non_withdrawable_stake(self) -> NU:
+        staked_amount: NuNits = self.staking_agent.non_withdrawable_stake(staker_address=self.checksum_address)
```

Comment:
Perhaps "locked stake"?

Reply:
I'm not sure about `locked stake` because we alredy have `locked` term and this term related to particular period, but this amount is what is not possible to withdraw and related to current and next period.

Reply:
Heh - sure.  A docstring will help.  There are many kinds of balance methods!

Reply:
✔️